### PR TITLE
Remove admin,debug,db api from exposed modules

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: geth/params/defaults.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 	ListenAddr = ":0"
 
 	// APIModules is a list of modules to expose via any type of RPC (HTTP, IPC, in-proc)
-	APIModules = "db,eth,net,web3,shh,personal,admin,debug"
+	APIModules = "db,eth,net,web3,shh,personal,debug"
```
Comment (adambabik) [inline geth/params/defaults.go]: I would remove `db`, `personal` and `debug` as well. If one needs them, they can be turned on in the config.

Comment (dshulyak) [reply geth/params/defaults.go]: i think personal_sign/personal_ecrecover is commonly used in dapps. let's leave it.
i will remove two others

Comment (mandrigin) [reply geth/params/defaults.go]: yep, personal should stay. it will require a confirmation from the client side anyway

**Review by mandrigin — APPROVED** [review]
Looks good!

**Review by adambabik — APPROVED** [review]

**Pull request merged**